dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/FourLetterWordMain.java

```diff
@@ -25,10 +25,7 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.OutputStream;
-import java.net.InetAddress;
-import java.net.InetSocketAddress;
-import java.net.Socket;
-import java.net.SocketTimeoutException;
+import java.net.*;
```

Comment:
Don't use import *.
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad